.github/workflows/commit-queue.yml

```diff
@@ -0,0 +1,76 @@
+---
+# This action requires the following secrets to be set on the repository:
+#   GH_USER_NAME: GitHub user whose Jenkins and GitHub token are defined below
+#   GH_USER_TOKEN: GitHub user token, to be used by ncu and to push changes
+#   JENKINS_TOKEN: Jenkins token, to be used to check CI status
+
+name: Commit Queue
+
+on:
+  schedule:
+    - cron: "*/5 * * * *"
```

Comment:
isn't there a pull request labelled event?

Reply:
There is, but when any event is triggered on a PR from a fork secrets are not available and GITHUB_TOKEN is read only, so we can't use ncu because we won't have access to the Jenkins token, we can't push because read only/no access to a personal token, and no changing labels or commenting or closing issues.

I tried to explain it in the implementation overview doc, but maybe we need to elaborate more on that? Or maybe also add a comment here explaining why it is implemented that way?

Reply:
Comment would be nice, yeah.